### PR TITLE
community/syslog-summary: new aport

### PR DESCRIPTION
**Pull request by itoffshore**
also includes it's dependent library: `py-magic`

### CODE REVIEW

File: community/syslog-summary/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+prepare() {
+        local i
+	cd "$builddir"
+        for i in $source; do
+                case $i in
+                *.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
+                esac
+        done
+}
```
Comment (jirutka) [inline community/syslog-summary/APKBUILD]: Please remove `prepare` function; applying of patches is already handled by the `default_prepare` function that is called when no custom `prepare` function is declared.

File: community/py-magic/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+pkgver=0.4.12
+pkgrel=0
+pkgdesc="A python wrapper for libmagic"
+url="https://pypi.python.org/pypi/python-magic/"
+arch="noarch"
+license="MIT"
+depends="python libmagic"
+makedepends="python2-dev python3-dev py-setuptools"
+install=""
+options="!strip"
```
Comment (jirutka) [inline community/py-magic/APKBUILD]: Why this package needs stripping to be disabled?

Comment (itoffshore) [inline community/py-magic/APKBUILD]: no point in stripping symbols from python scripts

Comment (jirutka) [inline community/py-magic/APKBUILD]: If it does not contain any binaries, then stripping is not performed anyway; I don’t see a point in explicit disable.

**Pull request closed without merging**